sim/hunter/runes.go

```diff
@@ -440,6 +434,27 @@ func (hunter *Hunter) applyHitAndRun() {
 	}
 }
 
+func (hunter *Hunter) applyCatlikeReflexes() {
+	if hunter.HasRune(proto.HunterRune_RuneHelmCatlikeReflexes) {
+		label := "Catlike Reflexes"
+		if hunter.HasAura(label) {
+			return
+		}
+		core.MakePermanent(hunter.RegisterAura(core.Aura{
+			Label: label,
+			OnInit: func(aura *core.Aura, sim *core.Simulation) {
+				hunter.AddStatDynamic(sim, stats.Dodge, 20*core.DodgeRatingPerDodgeChance)
+				if hunter.pet != nil {
+					hunter.pet.AddStat(stats.Dodge, 9*core.DodgeRatingPerDodgeChance)
+				}
+				if hunter.FlankingStrike != nil {
+					hunter.FlankingStrike.CD.Multiplier = 50
```

Comment:
We should modify this rather than assign
```suggestion
					hunter.FlankingStrike.CD.Multiplier -= 50
```